mava/utils/jax_training_utils.py

```diff
@@ -15,9 +15,73 @@ def action_mask_categorical_policies(
         distribution.logits,
         jnp.finfo(distribution.logits.dtype).min,
     )
+
     return tfd.Categorical(logits=masked_logits, dtype=distribution.dtype)
 
 
+def compute_running_mean_var_count(
+    stats: jnp.ndarray, batch: jnp.ndarray
+) -> jnp.ndarray:
+    """Updates the running mean, variance and data counts during training.
+
+    stats (array) -- mean, var, count.
```

Comment:
What is the purpose of the count param in stats? Like to prevent division by 0? If so maybe it should just be constant? 
That way we could make mean and var separate variables since passing in an array isn't always very readable.

Reply:
We need to it keep track of the total number of samples. So that every time when computing the running mean and variance we divide by the correct factor.